packages/camera/tizen/src/camera_device.cc

```diff
@@ -728,8 +761,8 @@ bool CameraDevice::SetPreviewFormat(CameraPixelFormat format) {
 
 bool CameraDevice::SetPreviewSize(Size size) {
   int w, h;
-  w = static_cast<int>(size.width);
-  h = static_cast<int>(size.height);
+  w = static_cast<int>(round(size.width));
```

Comment:
Why do we need this?

Reply:
The type of ```Size```'s memeber are double, I thought it was better than throwing away the decimal point.